charts/kueue/templates/_helpers.tpl

```diff
@@ -48,7 +48,7 @@ Selector labels
 {{- define "kueue.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "kueue.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
-app.kubernetes.io/component: controller
```

Comment:
Looking at the previous PR diff, it looks like this line can just be removed https://github.com/kubernetes-sigs/kueue/pull/1695/files#diff-cd784975dcc8020c6410c41d1869734647919b3d351c23006b1e4c41c3f580da

Reply:
However, the fact that you changed this and helm still worked might be an indication that helm does some extra steps when upgrading....

Can you try not changing this file and doing a manual upgrade using helm?

Reply:
Ohhh, I see, it was added here in the past https://github.com/kubernetes-sigs/kueue/pull/1695/files#diff-1fa2d9febb7e8abaca7e772caeada182ec33aed9a3b3fa09f0e56775cf02336eL78

Reply:
Yeah, leaving that line causes an error when upgrading